### PR TITLE
Fix Instance.new arguments

### PR DESCRIPTION
**Pull request by Minstrix**
Code_FkuLUC6VbK

### CODE REVIEW

**Comment by Jaguar-515** [comment]
https://github.com/Kampfkarren/selene/issues/114#issuecomment-637844358

Kampf stated in an old issue that he won't fix this as you **shouldn't** pass a parent argument in Instance.new().

On another note, could we make a lint for this? Seeing that sometimes this arises up occasionally, we should start to make a lint for this that allow/disallow using two arguments for Instance.new().

**Comment by Minstrix** [comment]
Ah, that's my bad then, I thought I'd searched the issues but must've only checked for past pull requests. Personally, I still don't think this should be marked as incorrect usage for no more than consistency. But, if Kampf is firm on this, making an allow/disallow lint might be a good compromise, although I'd have no idea how to go about doing that.

**Comment by Kampfkarren** [comment]
I'd rather make this its own lint, or buff the standard library lint to allow for cases like this.

File: selene/src/roblox/base.toml [hunk 1]
```diff
@@ -338,6 +338,9 @@ type = "..."
 [[Instance.new.args]]
 type = "string" # This is changed at generate time
 
+[[Instance.new.args]]
+type = "any"
```
Comment (Kampfkarren) [inline selene/src/roblox/base.toml:342]: This would break every case of `Instance.new("Class")` by the way, since you're not doing `required = false`.

**Pull request closed without merging**